digitalocean/database/resource_database_cluster_test.go

```diff
@@ -384,6 +384,34 @@ func TestAccDigitalOceanDatabaseCluster_WithVPC(t *testing.T) {
 	})
 }
 
+func TestAccDigitalOceanDatabaseCluster_WithBackupRestore(t *testing.T) {
+	var originalDatabase godo.Database
+	var backupDatabase godo.Database
+
+	originalDatabaseName := acceptance.RandomTestName()
+	backupDatabasename := acceptance.RandomTestName()
+
+	backUpRestoreConfig := fmt.Sprintf(testAccCheckDigitalOceanDatabaseClusterConfigWithBackupRestore, originalDatabaseName, backupDatabasename, originalDatabaseName)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
```

Comment:
adding the `CheckDestroy:      testAccCheckDigitalOceanDatabaseClusterDestroy` here for some reason results in a `cluster cannot be found`.

So naturally I figured it was a timing issue between the cluster being online and the call to preform a backup. So I broke up the calls in the test below and added a sleeper but that resulted in a `code = Internal desc = failed to list cluster backups before a fork attempt` [here](https://github.internal.digitalocean.com/digitalocean/cthulhu/blob/master/docode/src/do/teams/paas/dbaas/dbaas/server.go#L1707)

What ended up working consistently for this test was 1) removing the `checkDestory` func and 2) combining the create cluster config and backup restore config into one config.

Not sure why this behavior occurred...